### ISSUE TITLE
[Feature Request] Support multiple symbols

### ISSUE THREAD

**Issue by BradCandell** [Ebonsignori/obsidian-at-symbol-linking#28]
First off, this implementation is excellent! It was one of the primary reasons I left Microsoft OneNote in favor of Obsidian.

Like many, my primary use of the tooling is to refer to "People" or "Contacts" so I can "@" them just like many of today's more popular applications!

I have had a FEW different times where I've wanted to use that same style to link to another type of data. I can see the & or $ being something that people might have unique ideas to use.

The TLDR of this request is to provide the means to insert multiple symbols and configure them independently. I am a developer myself, and would be happy to look at contributing if this were something that you think the extension could benefit from having, but didn't necessarily have the time to implement yourself.

Cheers! I will find a way to properly send my gratitude for this extension! It really changed the game for me and my note taking!

**Comment by Ebonsignori** [comment]
Thank you <3, I also migrated to Obsidian from OneNote when I realized I could make this happen via a plugin.

Excellent idea. Makes me wish I'd simply named the plugin "symbol linking" instead 😁 

I'm actively working on this, but might be a few weeks until it's finished. 

Trying to make sure that @ symbol linking works as the default out of the box without ruining people's existing configurations when they upgrade

**Comment by sytone** [comment]
I just came looking for this as I wanted to link to projects using the + symbol and still be able to link to people using the @ symbol. Is there a branch where you are working on this I can try out?

**Comment by sytone** [comment]
@Ebonsignori any update on this work? Been quiet for a while and I am hoping you are still looking at this.

**Comment by Mara-Li** [comment]
Hello,
My fork allows doing that: https://github.com/Mara-Li/obsidian-symbol-linking.

**Comment by Ebonsignori** [comment]
@BradCandell @sytone completed in https://github.com/Ebonsignori/obsidian-at-symbol-linking/releases/tag/2.0.0 with the additional feature of specifying symbols to create new files in specific folders using specific templates
